packages/dcos-integration-test/extra/test_tweeter_demo.py

```diff
@@ -0,0 +1,95 @@
+import logging
+import os
+import time
+
+import pytest
+import requests
+
+import test_helpers
+
+from test_packaging import (
+    skipif_insufficient_resources
+)
+
+log = logging.getLogger(__name__)
+
+DEMO_RESOURCE_REQUIREMENTS = {
+    'number_of_nodes': 5,
+    'node': {
+        'disk': 0,
+        'mem': 0,
+        'cpus': 1
+    }
+}
+
+
+def _get_tweeter_app_definition():
+    response = requests.get('https://raw.githubusercontent.com/mesosphere/tweeter/master/tweeter.json')
+    return response.json()
+
+
+def _get_post_tweets_app_definition():
+    response = requests.get('https://raw.githubusercontent.com/mesosphere/tweeter/master/post-tweets.json')
+    return response.json()
+
+
+def _get_tweet_count(address):
+    """Count the number of tweets on the tweeter app's webpage
+
+    Args:
+        address: str URL including http://
+    """
+    response = requests.get(address)
+    return response.text.count('tweet-content')
+
+
+@pytest.skipif(
+    test_helpers.expanded_config.get('security') == 'strict',
+    reason='This test requires specific service accounts if security is strict.')
```

Comment:
This reasoning statement is little confusing to me.

> 'This test requires specific service accounts if security is strict.'

This is `pytest.skipif security == strict`

So, you want this test not be run in strict mode. Isn't it? 

Could the reason be written as

> This test is not-applicable for strict mode.

Reply:
It is still run in strict mode, but there's a separate enterprise test for it (WIP) that uses the service-accounts secrets stuff with marathon-lb.